sopel/plugins/rules.py

```diff
@@ -591,8 +594,8 @@ def get_output_prefix(self):
             See the :class:`sopel.bot.SopelWrapper` class for more information
             on how the output prefix can be used.
         """
-        raise NotImplementedError
 
+    @abc.abstractmethod
     def match(self, bot, pretrigger):
```

Comment:
No type hint?

Reply:
It's a more complex type hint, so I'll do that in a future PR (when I'll be *real* serious about type hint).

Reply:
Actually, I'm going to be a bit lazy and just do `-> Iterable[Any]`.